src/Transaction/Tx.js

```diff
@@ -64,7 +64,7 @@ class Tx {
             this.Tx["Vout"] = tx["Tx"]["Vout"];
             this.Tx["Fee"] = tx["Tx"]["Fee"];
             for (let i = 0; i < tx["Tx"]["Vin"].length; i++) {
-                let cTx = await madWallet.Rpc.getMinedTransaction(vin[l]["TXInLinker"]["TXInPreImage"]["ConsumedTxHash"])
```

Comment:
The code here looks like a typo which also throws an eslint error so it's fixed in the line below.